webpack/assets/javascripts/react_app/components/Layout/components/UserDropdowns.js

```diff
@@ -47,6 +48,14 @@ const UserDropdowns = ({
                 </MenuItem>
               )
             )}
+            <MenuItem key="user_dropdown_divider" divider />
+            <MenuItem
+              key="user_logout"
+              className="user_menuitem"
+              onClick={() => onLogout()}
+            >
+              {__('Log Out')}
+            </MenuItem>
```

Comment:
Instead of adding this menu item separately from all the rest, could we just modify the existing item to pass `onClick` instead of `html` attribute with a small function that sends post request to its url instead of get?